src/js/App/GlobalFilter/GlobalFilter.js

```diff
@@ -170,8 +170,13 @@ const GlobalFilter = () => {
             }
             setIsOpen(false);
           }}
-          onApplyTags={(selected) => {
-            setValue(() => selected.reduce((acc, { namespace, key, value }) => updateSelected(acc, namespace, key, value, true), selectedTags));
+          onApplyTags={(selected, sidSelected) => {
+            setValue(() =>
+              [...selected, ...sidSelected]?.reduce(
```

Comment:
This spread will always produce at least an empty array. So no need for the `?`. I would be more worried about `selected` or `sidSelected` being `undefined`. If that is the case can we add a default value `[]` to the arguments?